src/write-batch.js

```diff
@@ -490,9 +510,7 @@ class WriteBatch {
       request.transaction = explicitTransaction;
     }
 
-    // We store the number of operations that we sent as part of this commit
-    // as the user is able to add additional operations afterwards.
-    let opCount = this._writes.length;
+    this._committed = true;
```

Comment:
It might be worth adding a comment (to commit() or the top of commit_() calling out that we're explicitly not calling verifyNotCommitted() because we want to let people retry committing batches.  Else somebody might come along and assume it's an oversight and "fix" it. :-)

Reply:
Done.